src/test/resources/integration/multi-module-install-plugin-2/pom.xml

```
@@ -73,7 +73,7 @@
                             </deployProperties>
                             <artifactory>
                                 <includeEnvVars>true</includeEnvVars>
-                                <envVarsExcludePatterns>*password*,*secret*,*key*,*token*,*passphrase*
+                                <envVarsExcludePatterns>*username*,*password*,*secret*,*key*,*token*,*passphrase*
```

I notice that you have `username` and `password` properties in the package.xml. This might be a good opportunity to make sure that the username appears in the build-info.
```suggestion
                                <envVarsExcludePatterns>*password*,*secret*,*key*,*token*,*passphrase*
```